terrenas/simple_script.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+from terrenas.terrenas import single_iteration_HDF5
+import numpy as np
+
+group = int(sys.argv[1])
+
+seed_array = np.arange(10**6).reshape(10**3,10**3)
+
+output_path = os.getenv('SCRATCH') + '/training_data/'
+kwargs_observing = {}
+kwargs_sample_redshifts = {'zd_min':0.5, 'zd_max':0.5,
+                     'zs_min':1.5, 'zs_max':1.5}
+
+kwargs_sample_macromodel = {'theta_E_low':0.8, 'theta_E_high':1.2,
+                      'q_low':0.75, 'q_high':0.75,
+                      'log10_shear_mag_low':np.log10(0.05), 'log10_shear_mag_high':np.log10(0.05),
+                      'gamma_epl_low':1.9, 'gamma_epl_high':2.1,
+                      'a4_mean':0.0, 'a4_variance':0.0}
```

Comment:
does having zero variance cause an error when sampling the Gaussian?

Reply:
Not in Numpy, as far as I know.